### PR TITLE
About screen

### PR DESCRIPTION
**Pull request by subdigital**
Builds on #32, adds a footer to the issue container vc (so it's "sticky").

### CODE REVIEW

**Comment by alexargo** [comment]
Should be much nicer now and way more native. Let me know if you want me to tweak anything.

File: FiveCalls/FiveCalls/AboutHtmlViewController.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+    
+    func webView(_ webView: UIWebView, shouldStartLoadWith request: URLRequest, navigationType: UIWebViewNavigationType) -> Bool {
+        switch navigationType {
+        case .linkClicked:
+            // Open links in Safari
+            guard let url = request.url else { return true }
+            
+            if #available(iOS 10.0, *) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
```
Comment (mccarron) [inline FiveCalls/FiveCalls/AboutHtmlViewController.swift]: Those second two parameters have defaults so you can do just `UIApplication.shared.open(url)`

Comment (subdigital) [inline FiveCalls/FiveCalls/AboutHtmlViewController.swift]: I'll update this.

**Comment by subdigital** [comment]
This looks great!  I'm going to tweak a few things and then get this merged in.  Thanks, @alexargo!

**Pull request merged**